conf/defaults.yaml

```diff
@@ -231,16 +228,13 @@ topology.multilang.serializer: "org.apache.storm.multilang.JsonSerializer"
 topology.shellbolt.max.pending: 100
 topology.skip.missing.kryo.registrations: false
 topology.max.task.parallelism: null
-topology.max.spout.pending: null
+topology.max.spout.pending: null  # TODO: We dont need this any more
 topology.state.synchronization.timeout.secs: 60
-topology.stats.sample.rate: 0.05
+topology.stats.sample.rate: 0.001
```

Comment:
Likewise I said some frameworks even don't sample and measure all for meters. JStorm is one of example, and they (Alibaba) said "every day the JStorm Cluster handles 1.5 PB/2 Trillion messages" in 2015, which is 23,000,000 messages per seconds. Yes we don't know how many servers they use, but they also claim their performance in website http://jstorm.io:8080/Performance/ and metrics seems not make considerable bottleneck. 

One thing to consider is that current metrics implementation may be really slower than dropwizard thing, so we may want to adjust rate if we believe it is really a bottleneck.

Btw, most of the cases users would want to have operational convenience over performance. We should consider first what users are facing day by day.